paths_cli/wizard/load_from_ops.py

```diff
@@ -25,24 +23,21 @@ def _get_ops_storage(wizard):
         storage = INPUT_FILE.get(filename)
     except Exception as e:
         wizard.exception(FILE_LOADING_ERROR_MSG, e)
-        return
+        return None
 
     return storage
 
+
 @get_object
 def _get_ops_object(wizard, storage, store_name, obj_name):
-    name = wizard.ask(f"What's the name of the {obj_name} you want to "
-                      "load? (Type '?' to get a list of them)",
-                      helper=named_objs_helper(storage, store_name))
-    if name:
-        try:
-            obj = getattr(storage, store_name)[name]
-        except Exception as e:
-            wizard.exception("Something went wrong when loading "
-                   f"{name}. Maybe check the spelling?", e)
-            return
-        else:
-            return obj
+    store = getattr(storage, store_name)
```

Comment:
this can raise an `AttributeError that should probably be handled

Reply:
I think the `AttributeError` here could only occur in bad calling code (i.e., not from Wizard users, but possibly from plugin code.) Is there a reasonable way to recover from that here? I don't see one; I think think raising the `AttributeError` would be the right thing to do.

(Example where this might occur: say plugin code said the object was supposed to be found in `storage.eeeeengines`. Seems to me like this is an unrecoverable error in the plugin, no?)

Reply:
yeah, seems like it. I just thought about it because the old code in this file did seem to handle this error by accident (It seems like it was only intended to handle mistyping of the `object`, not the actual storage name).

Reply:
Yeah, you're correct that the old code would have caught and handled that exception. I think it also would have put the user in an infinite loop, since there's no way from the user end to change `store_name`. I hadn't noticed that before, but definitely another reason in support of this change!

The main purpose of here was to change the prompt from basically: "What's the name of the engine you want to load from storage?" (implicit -- you should have this memorized) to something that is implicitly more "Here are the engines in that file; which one do you want?", allowing selection by number or name. I think that's a better UX, and it allows all the "retry on bad input" stuff to be handled by the `@get_object` decorator.